docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -176,67 +191,62 @@ After a failover, do the following:
 
-1. Stop the HA synchronization. This ensures that only backups of the original YugabyteDB Anywhere version are synchronized to the standby instance.
-1. [Upgrade the active instance](../../upgrade/). Expect a momentary lapse in availability for the duration of the upgrade.
+## Remove a standby instance
 
-    If the upgrade is successful, proceed to step 3.
+To remove a standby instance from a HA cluster, you need to remove it from the active instance's list, and then delete the configuration from the instance to be removed, as follows:
 
-    If the upgrade fails, perform the following:
+1. On the active instance's list, click **Delete Instance** for the standby instance to be removed.
 
-    - Decommission the faulty active instance in the active-standby pair.
-    - [Promote the standby instance](#promote-a-standby-instance-to-active).
-    - Do not attempt to upgrade until the root cause of the upgrade failure is determined.
-    - Delete the HA configuration and bring up another standby instance at the original YugabyteDB Anywhere version and reconfigure HA.
-    - After the root cause of failure has been established, repeat the upgrade process starting from step 1. Depending on the cause of failure and its solution, this may involve upgrading to a different version of YugabyteDB Anywhere.
+1. On the standby instance you wish to remove from the HA cluster, on the **Admin > High Availability** tab, click **Delete Configuration**.
 
-1. On the upgraded instance, perform post-upgrade validation tests that may include creating or editing a universe, backups, and so on.
-1. [Upgrade the standby instance](../../upgrade/).
-1. Enable HA synchronization.
-1. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade.
+The standby instance is now a standalone instance again.
 
-The following diagram provides a graphical representation of the upgrade procedure:
+After you have returned a standby instance to standalone mode, the information on the instance is likely to be out of date, which can lead to incorrect behavior. It is not recommended to continue to use this standby instance for any management operations. Uninstall YBA from this instance and reinstall it to return it to a clean state before using it as a standalone instance.
 
-![High availability upgrade](/images/yp/high-availability/ha-upgrade.png)
+## Monitoring
 
-Where Version A is the original YugabyteDB Anywhere version present on the active and standby instances, and Version B is the version you are upgrading to.
+The easiest way to determine the health of your HA configuration is to monitor the overall HA state on your active YBA instance. The overall HA state is computed from the individual instance states, and can be viewed on the **Instance Configuration** page.
```

Comment:
which can be viewed

Reply:
And also highlight Global State is viewed on Replication Configuration tab, unlike the individual instance states.